### ISSUE TITLE
Task3: MMLIST Dataset Classification

### ISSUE THREAD

**Issue by adidonner** [adidonner/Machine-Deep-Leap-learning-Python-Based#1]
Clothing collection classification task
We took a labeled set of a clothing collection and built a recognition model written in Python using ChatGPT (C) that receives an image and knows how to classify to which of the 10 categories of clothing (shoes, shirt, etc.) the image belongs
I built a neural network architecture. And I tried to check how many layers and how many perspektions in each layer, I guessed and then I tried to train the network.
The process:
1. Transferring an umbrella to a vector using FLATTEN
2. Normalize the values from 0-255 to 0-1 to make the calculation easier
3. On the last side of the network there were 10 exits where it was determined for each category what the probability is that the image is from this category.
4. The computer presented as an answer the category in which the probability was the highest
5. We built 6 models in which different parameters were chosen for the architecture:
5.1 BATCH - the group of samples after which it calculates error and updates weight
5.2 EPOCH - Go over all the samples (how many items are there in the data)
5.3 LEARNING RATE - step size, how much it changes the weights
משימת סיווג קולקצית בגדים
לקחנו סט מתויג של קולקצית בגדים ובנינו באמצעותו מודל  זיהוי שנכתב בשפת פייתון בעזרת  ChatGPT (C)  שמקבל תמונה ויודע לסווג לאיזה מ-10 הקטגוריות של בגדים (נעליים, חולצה וכו') שייכת התמונה
בניתי ארכיטקטורה  של רשת נוירונים. וניסיתי לבדוק בכמה שכבות וכמה פרספקטונים בכל שכבה, נחשתי ואז ניסיתי לאמן את הרשת.
התהליך:
1. העברת מטריצה לוקטור בעזרת FLATTEN
 נירמול הערכים מ 0-255 ל 0-1 כדי להקל על עבודת החישוב
2. בצידה האחרון של הרשת היו 10 יציאות בהם נקבע לכל קטגוריה מה ההסתברות שהתמונה היא מקטגוריה זו.
3. המחשב הציג כתשובה את הקטגוריה בה ההסתברות היתה הכי גבוהה 
4. ניבנו 6 מודלים בהם היה נבחרו פרמטרים שונים לארכיטקטורה: 
       4.1 BATCH -מקבץ הסמפלים שאחריו הוא מחשב שגיאה ומעדכן משקל
       4.2 EPOCH - מעבר על כל הסמפלים (כמה פריטים יש בדאטה)
       4.3  LEARNING RATE - גודל צעד, כמה הוא משנה את המשקלים

Conclusion: Adding layers did not improve CROSS ENTROPHY of 12% error but improved its "confidence" in the answer, meaning the probability of choosing a certain category from among others was closer to 1.
הוספת שכבות לא שיפרה את CROSS ENTROPHY מ 12% טעות אך שיפרה את "הביטחון" שלו בתשובה כלומר ההסתברות לבחירה קטגוריה מסוימת מתוך אחרות היתה בשיעור קרוב יותר ל-1.
